src/routes/chatbot/+page.svelte

```diff
@@ -0,0 +1,137 @@
+<script>
+	import { writable } from 'svelte/store';
+
+	let message = '';
+	let messages = writable([]);
+	let botResponses = [
+		'Hallo! Welkom bij de Openbare Bibliotheek Amsterdam. Waarmee kan ik u vandaag helpen?',
+		'Dat klinkt goed! Kunt u mij de titel of het onderwerp geven waarin u geïnteresseerd bent?',
+		'Ik kan daar zeker bij helpen! Zoekt u naar een specifiek boek of wilt u een lijst van aanbevelingen?',
+		'Dat boek is te vinden op de tweede verdieping, bij de sectie Literatuur. Wilt u nog iets anders weten?',
+		'Graag gedaan! Als u nog andere vragen heeft, aarzel niet om contact op te nemen. Fijne dag verder en we hopen u snel weer te zien in de OBA!'
+	];
+
+	function sendMessage() {
+		messages.update((currentMessages) => {
+			let updatedMessages = [...currentMessages, { text: message, sender: 'user' }];
+			message = '';
+
+			setTimeout(() => {
+				let botResponseIndex = currentMessages.length / 2;
+				let botResponse = botResponses[botResponseIndex];
+				messages.update((currentMessages) => [
+					...currentMessages,
+					{ text: botResponse, sender: 'bot' }
+				]);
+			}, 1000);
+
+			return updatedMessages;
+		});
+	}
+</script>
+
+<main>
+	<div class="chatbox">
+		{#each $messages as msg (msg.text)}
+			<div class={msg.sender}>
+				<p>{msg.text}</p>
+			</div>
+		{/each}
+	</div>
+
+	<input
+		bind:value={message}
+		on:keydown={(e) => {
+			if (e.key === 'Enter') {
+				e.preventDefault();
+				sendMessage();
+			}
+		}}
+		placeholder="Type a message..."
+	/>
+	<button on:click={sendMessage}>Send</button>
```

Comment:
Hier een formulier omheen zetten?
Dan kan je de chat PE opzetten met SvelteKit Form Actions: 

https://kit.svelte.dev/docs/form-actions

Reply:
Ik zal dit zeker doen! Dit was meer een chatbot V1 waarin de focus erop lag dat het werkte. Ik zal in V2 hier meer op letten